pelican/tests/test_importer.py

```diff
@@ -420,17 +420,17 @@ def test_attachments_associated_with_correct_post(self):
                 expected = ('https://upload.wikimedia.org/wikipedia/commons/'
                             'thumb/2/2c/Pelican_lakes_entrance02.jpg/'
                             '240px-Pelican_lakes_entrance02.jpg')
-                self.assertEqual(self.attachments[post][0], expected)
+                self.assertEqual(self.attachments[post], {expected})
```

Comment:
Minor quibble: since the variable is called `expected`, I would prefer if it already were the expected set, i.e. `expected = {'https://...'}`.

Reply:
Sorted